scipy/stats/tests/test_distributions.py

```diff
@@ -1496,6 +1512,15 @@ def test_fit_warnings(self):
         assert_raises(FitDataError, stats.pareto.fit, [5, 2, 3], floc=1,
                       fscale=3)
 
+    def test_negative_data(self, rng):
+        data = stats.pareto.rvs(loc=-130, b=1, size=100, random_state=rng)
+        assert np.all(data < 0)
```

Comment:
Isn't `assert_array_less()` preferable here? It should give a better indication of mismatch quantification as opposed to a plain `AssertionError`:

```python
In [2]: from numpy.testing import assert_array_less

In [3]: arr = np.array([-1, -2, 1])

In [4]: assert np.all(arr < 0)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-4-59177056fc65> in <module>
----> 1 assert np.all(arr < 0)

AssertionError: 

In [5]: assert_array_less(arr, 0)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-5-f3d944fb1e74> in <module>
----> 1 assert_array_less(arr, 0)

    [... skipping hidden 1 frame]

~/python_venvs/python_395/lib/python3.9/site-packages/numpy/testing/_private/utils.py in assert_array_compare(comparison, x, y, err_msg, verbose, header, precision, equal_nan, equal_inf)
    842                                 verbose=verbose, header=header,
    843                                 names=('x', 'y'), precision=precision)
--> 844             raise AssertionError(msg)
    845     except ValueError:
    846         import traceback

AssertionError: 
Arrays are not less-ordered

Mismatched elements: 1 / 3 (33.3%)
Max absolute difference: 2
Max relative difference: inf
 x: array([-1, -2,  1])
 y: array(0)
```

Reply:
Great improvement, thanks! I'm going to add it to a review in case there are other comments so I can add it in batch.

Reply:
Done!